includes/internal/emails/class-email-sender.php

```diff
@@ -200,27 +211,47 @@ private function get_email_post_by_name( $email_identifier ) {
 
 		return $email_post;
 	}
-
 	/**
 	 * Get the email body rendered in the email template.
 	 *
-	 * @param string $email_content The placeholder replaced email content.
+	 * @param array $placeholders The placeholder replaced email content.
 	 *
 	 * @return string
 	 */
-	private function get_templated_post_content( $email_content ) {
+	private function get_templated_post_content( $placeholders ) {
 		global $sensei_email_data;
-
-		$sensei_email_data['email_body'] = $email_content;
 		$sensei_email_data['body_class'] = '';
 
+		// Force use the default template usage.
+		$template = do_blocks( get_block_template( Email_Page_Template::ID, 'wp_template' )->content );
+
+		$post_content = $this->replace_placeholders(
+			$template,
+			$placeholders
+		);
+
+		$post_content                    = $this->add_base_url_for_images( $post_content );
+		$sensei_email_data['email_body'] = $post_content;
+
 		ob_start();
 
 		require dirname( __FILE__ ) . '/../../../templates/emails/block-email-template.php';
 
 		return ltrim( ob_get_clean() );
 	}
 
+	/**
+	 * Append the site URL on all images before send the email.
+	 *
+	 * @param string $content The email content that should be updated.
+	 *
+	 * @return string
+	 */
+	private function add_base_url_for_images( $content ) {
+		$image_path = '/wp-content/plugins/sensei/assets/dist/images/';
```

Comment:
I think this assumption might be wrong in some cases :)
Better use a constant (or a relative path).

Reply:
New try here 627d5cea1c18c3e021b005729b2deec235b063c0
Could you please test it?

Reply:
I think you can use `plugin_dir_url( SENSEI_LMS_PLUGIN_FILE )` for this purpose.
And maybe we have a constant for it.

Reply:
I found `Sensei()->plugin_url` (`includes/class-sensei.php:67`)